index.html

```diff
@@ -6,6 +6,60 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    <section id="exercise-one">
+        <button id="background-changing-button">Change the color!</button>
+        <input id="color-input" />
+    </section>
+    <br>
+    <hr>
+    <br>
+    <section id="exercise-two">
+        <input id="font-size-input" type="number" />
+        <p id="font-size-paragraph">Look - I can change size!</p>
+    </section>
+    <br>
+    <hr>
+    <br>
+    <section id="exercise-three">
+        <button id="color-changing-button">Pick a side in the House of the Dragon conflict! </button>
+        <p class="black-paragraph">Dragon-One</p>
+        <p class="black-paragraph">Dragon-Two</p>
+        <p class="black-paragraph">Dragon-Three</p>
+    </section>
+    <br>
+    <hr>
+    <br>
+    <section id="exercise-four">
+        <input id="text-input">
+        <button id="text-changing-button">Click</button>
+    </section>
+    <br>
+    <hr>
+    <br>
+    <section id="exercise-five">
+        <button class="yellow-button">Red or yellow?</button>
```

Comment:
Why did you use a class here? Do you plan on having multiple buttons like that?

Reply:
Because 99% of time i've used classes in the past to style something. Look at my CV or other projects with css.
Thought it's better. Only if there are more than one buttons, or something is styled in CSS i used classes instead of ID's.